### PR TITLE
Press Enter to Send

### PR DESCRIPTION
**Pull request by JessZanewich**
This allows a user to press enter to send a message, rather than having to click the send button.

### CODE REVIEW

File: src/chat-app-frontend/src/Components/WebSocketChatBox.tsx [hunk 1]
```diff
@@ -57,6 +57,12 @@ export class WebSocketChatBox extends Component<WebSocketWrapperProps, WebSocket
 		console.log(this.state.clientMessage);
 	}
 
+	onKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+		if(event.keyCode === 13) {
```
[the remark this answers is not in the document]
Comment (JessZanewich) [reply src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]: Are you talking for the key code?

[the remark this answers is not in the document]
Comment (MostlyArmless) [reply src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]: yeah

File: src/chat-app-frontend/src/Components/WebSocketChatBox.tsx [hunk 2]
```diff
@@ -57,6 +57,12 @@ export class WebSocketChatBox extends Component<WebSocketWrapperProps, WebSocket
 		console.log(this.state.clientMessage);
 	}
```
Comment (christoffsen) [inline src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]:
```suggestion
const ENTER_KEY_CODE = 13;
```

File: src/chat-app-frontend/src/Components/WebSocketChatBox.tsx (hunk 1 above)
Comment (christoffsen) [inline src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]:
```suggestion
		if(event.keyCode === ENTER_KEY_CODE) {
```

File: src/chat-app-frontend/src/Components/WebSocketChatBox.tsx [hunk 3]
```diff
@@ -57,6 +58,12 @@ export class WebSocketChatBox extends Component<WebSocketWrapperProps, WebSocket
 		console.log(this.state.clientMessage);
 	}
 
+	onKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+		if(event.keyCode === 13) {
```
Comment (christoffsen) [inline src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]:
```suggestion
		if(event.keyCode === ENTER_KEY_CODE) {
```

Comment (christoffsen) [inline src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]:
```suggestion
		if(event.keyCode === ENTER_KEY_CODE) {
```

Comment (christoffsen) [reply src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]: This should be right

File: src/chat-app-frontend/src/Components/WebSocketChatBox.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (JessZanewich) [reply src/chat-app-frontend/src/Components/WebSocketChatBox.tsx]: So just add a:
`const enterKey = 13;`

**Comment by JessZanewich** [comment]
Need to figure out how this fits in for future work.

**Pull request merged**